db/migration/1738574434234-AlterVariablesIndexes.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AlterVariablesIndexes1738574434234 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // Drop the unique index on shortName (and datasetId)
```

Comment:
nit: could make it a teeny bit clearer
```suggestion
        // Drop the unique index on (shortName ,datasetId)
```